src/components/Timer/index.js

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
-import './styles.css';
+import PropTypes from 'prop-types';
+import './styles.scss';
 
-const Timer = () => {
-    
+const Timer = ({ expirationDate }) => {
+    return (
+        <div>
+            <img src="https://placehold.co/20x20" alt="Timer" />
+            <h4>{expirationDate}</h4>
+        </div>
+    );
+};
```

Comment:
Since we properly haven't discussed how implement the Timer component, for now I'm working it this way.